src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseFileController.cs

```diff
@@ -60,6 +69,14 @@ public async Task<ActionResult<IEnumerable<FileInfo>>> GetAncillaryFiles(Guid re
                 .HandleFailuresOrOk();
         }
 
+        [HttpGet("release/{releaseId}/ancillary/{fileId}")]
+        public async Task<ActionResult<FileInfo>> GetFile(Guid releaseId, Guid fileId)
```

Comment:
I think there's a bit of a naming problem here although not sure what to suggest as the solution.

The route here is `/ancillary/` implying that it can only get ancillary files but the name is `GetFile` and the method is generic so sure enough, supplying the id of any file will return it.

The equivalent for getting this file 'info' for data files includes the term `FileInfo` like `GetDataFileInfo` in  the controller and `GetInfo` in the service.

I see that you can't use `/file/` in the route because it would conflict with the `Stream` method.

Maybe we need to think about the naming at a wider level.

Reply:
Talked to Ben about this - we decided to rename the existing `release/{releaseId}/file/{id}` endpoint to stream files to `release/{releaseId}/file/{id}/download`, allowing the new endpoint to use the original.